api/buildSrc/src/main/groovy/org/pmiops/workbench/tooling/GenerateAPIListingTask.groovy

```diff
@@ -0,0 +1,59 @@
+package org.pmiops.workbench.tooling
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import groovy.util.logging.Slf4j
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+/**
+ * See https://precisionmedicineinitiative.atlassian.net/browse/PD-2275
+ * Run this gradle task from the command line in the /api directory:
+ *
+ * `gradle listProjectAPIs`
+ *
+ * This generates a tab separated list of apis and descriptions that can
+ * then be easily pasted into a spreadsheet.
+ */
+@SuppressWarnings(["unused", "GrMethodMayBeStatic"])
+@Slf4j
+class GenerateAPIListingTask extends DefaultTask {
+
+    @TaskAction
+    void defaultAction() {
+        List<String> methods = ["get", "delete", "patch", "post", "put"]
```

Comment:
There could be, but in our case, there aren't, but this is just a convenience to quickly find the thing we want inside of an api. Looking over this, I think I see a bug. If there is a get and a post for the same api, this will find only one of them ... thanks.